apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/db/ShardRecoveryDaoJdbcImpl.java

```diff
@@ -144,7 +145,8 @@ public long saveShardRecovery(Connection con, ShardRecovery recovery) {
         try (PreparedStatement pstmt = con.prepareStatement(
                 "INSERT INTO shard_recovery(" +
                         "state, object_name, column_name, last_column_value, processed_object, updated, height) " +
-                        "VALUES (?, ?, ?, ?, ?, CURRENT_TIMESTAMP(), ?)"
+                        "VALUES (?, ?, ?, ?, ?, CURRENT_TIMESTAMP(), ?)",
```

Comment:
Does New h2 version not return generated keys without additional parameter?

Reply:
nope